app/assets/javascripts/index/new_note.js

```diff
@@ -130,11 +127,14 @@ OSM.NewNote = function (map) {
     });
 
     newNoteMarker.on("dragstart dragend", function (a) {
-      newHalo(newNoteMarker.getLatLng(), a.type);
+      removeHalo();
+      if (a.type !== "dragstart") {
```

Comment:
Why not check for the type being `dragend` which would be much easier to reason about than having to find the set of captured events and then remove `dragstart` from that to work out when this is going to trigger?

Reply:
halo is supposed to reappear on drag end, so I guess it makes more sense to check for `dragend`. Updated.